src/components/app.jsx

```diff
@@ -186,6 +220,24 @@ const App = (props) => {
           const color = await getDominantColor(collection.coverArt)
           setDominantColor({ primary: color, secondary: shadeColor(color, -20) })
         }
+      } else if (requestType === RequestType.COLLECTIBLES) {
+        let collectibleData
+        if (request.collectibleId) {
+          collectibleData = await getCollectible(request.handle, request.collectibleId)
+        } else {
+          collectibleData = await getCollectibles(request.handle)
+        }
+
+        if (!collectibleData) {
+          setDid404(true)
+          setCollectiblesResponse(null)
+        } else {
+          setDid404(false)
+          setCollectiblesResponse(collectibleData)
+          // TODO: Figure out how to record this
```

Comment:
follow up? https://github.com/AudiusProject/bedtime/blob/1167d83a39c55a58e310bda0eb2afbb6f80f4057/src/analytics/analytics.js#L15

you can just call this with whatever as long as it's consistent

`recordOpen(request.collectibleId, collectibleData.name, request.handle, request.url)`